package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "graph-nmap",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "cat test/nmap-out.xml | ts-node ./src/index.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@jupiterone/jupiter-managed-integration-sdk": "^33.6.0",
```

Comment:
This seems like a rather heavy dependency for the one function of checking a public IP address 🤔 Perhaps there is a smaller NPM package that does that kind of thing?

Reply:
i'm hoping to use this in a future iteration to create the Host entities (and potentially other entities)